src/test/java/org/biouno/unochoice/TestParametersOrder.java

```diff
@@ -60,12 +60,18 @@ public void testParametersOrder() {
         parameters.put("B", "B");
         parameters.put("A", "A");
 
+        System.out.println("D1");
+        /*
         ChoiceParameter parameter = new ChoiceParameter(
                 "script001", "description", "random name", new GroovyScript(SCRIPT, FALLBACK_SCRIPT),
-                ChoiceParameter.PARAMETER_TYPE_MULTI_SELECT, true);
+                ChoiceParameter.PARAMETER_TYPE_MULTI_SELECT, true, 0);
+
+        System.out.println("D2");
```

Comment:
We don't need to print anything in the console. And I think there was a problem with indentation below here too